src/main/java/de/szut/lf8_project/lf8/project/ProjectController.java

```diff
@@ -3,36 +3,38 @@
 import de.szut.employees.EmployeeAPI;
 import de.szut.employees.dto.EmployeeResponseDTO;
 import de.szut.lf8_project.exceptionHandling.InvalidDataException;
+import de.szut.lf8_project.lf8.timemanagement.dto.AddEmployeeToProjectDto;
 import de.szut.lf8_project.lf8.project.dto.CreateProjectDto;
 import de.szut.lf8_project.lf8.project.dto.GetProjectDto;
+import de.szut.lf8_project.lf8.timemanagement.TimeManagementEntity;
+import de.szut.lf8_project.lf8.timemanagement.TimeManagementService;
 import de.szut.lf8_project.utils.HTTPCodes;
+import de.szut.lf8_project.utils.HelperFunctions;
 import de.szut.lf8_project.utils.MediaTypes;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.media.Content;
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.AllArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
 import java.io.IOException;
-import java.text.MessageFormat;
-import java.util.*;
+import java.util.HashMap;
+import java.util.List;
 import java.util.stream.Collectors;
 
+@AllArgsConstructor
 @RestController
 @RequestMapping(value = "lf8/project")
 public class ProjectController {
+    private final TimeManagementService tmService;
```

Comment:
name is not clear, if someone has to look at this variable from anywhere else besides the location where it's defined they wouldn't have a clue what this is.